README.md

```diff
@@ -3,3 +3,68 @@
 The goal of this wiki is to modernize the information found in the long-standing OSDev wiki found at [wiki.osdev.org](https://wiki.osdev.org) and to decentralize it. It is currently **WIP** and **not ready** for use.
 
 This wiki is not endorsed by, does not have any connection to, and is not related in any way with another wiki found at [wiki.osdev.org](https://wiki.osdev.org).
+
+# Contributing
+
+## Installing dependencies
+
+This wiki is written in Haskell.
+
+The project has some non-Haskell dependencies. To install these, you will need [Python](https://www.python.org/downloads/), [Ruby](https://www.ruby-lang.org/fr/documentation/installation/) and [Rust](https://www.rust-lang.org/tools/install) installed:
```

Comment:
One of these links is French.

I also feel like instructing the user on how to get runtimes is unnecessary. All of the other dependencies are distinct projects with their own dependencies, so it should be left up to them to instruct on obtaining them.